.travis.yml

```diff
@@ -52,7 +52,7 @@ matrix:
 
 before_install:
   - pyenv global system 3.5
-  - pip3 install 'meson>=0.42,!=0.43'
+  - pip3 install 'meson>=0.42,!=0.43,!=0.44'
```

Comment:
`pip install 'meson==0.42'` would be a lot safer ...

Reply:
That's true, I was also thinking about that. I guess the idea behind that was to not forget about this issue and enforce getting back to the latest release at some point. To me it would be fine to keep it like this for another version, but then I'd also rather like to go with `==0.42` to avoid this becoming a regular distraction.